src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -14,15 +14,19 @@ export type ThemeProviderProps = {
    * Optional children.
    */
   children?: React.ReactNode;
+  /**
+   * Optional document specification to support popup windows.
+   */
+  owningDocument?: Document;
```

Comment:
I think this prop should be called `ownerDocument` to match the API (https://developer.mozilla.org/en-US/docs/Web/API/Node/ownerDocument)

Reply:
Unless there was a reason for this name that I'm not aware of?

Reply:
renamed